Code/Tools/SceneAPI/SceneCore/DataTypes/GraphData/ICustomPropertyData.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
```

Comment:
shouldn't the pragma once be after the legal header?

Reply:
oddly, that is how all the other interface headers had it so I kept it the same (such as Code\Tools\SceneAPI\SceneCore\DataTypes\GraphData\ITransform.h)

Reply:
huh, but not Code\Tools\SceneAPI\SceneCore\DataTypes\GraphData\IMeshVertexTangentData.h so I will put it after the legal header